.github/workflows/label.yml

```diff
@@ -0,0 +1,15 @@
+name: Labeler
+on: [pull_request_target]
+
+jobs:
+  label:
+
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+
+    steps:
+    - uses: actions/labeler@v4
```

Comment:
Ensure that the permissions are set correctly to avoid any security issues, especially with 'pull-requests: write'.